crates/types/src/entities/relayer/message.rs

```diff
@@ -273,40 +273,9 @@ impl MessageProof {
 }
 
 /// Represents the status of a message
-pub struct MessageStatus {
-    /// The message state
-    pub state: MessageState,
-}
-
-impl MessageStatus {
-    /// Constructor for `MessageStatus` that fills with `Unspent` state
-    pub fn unspent() -> Self {
-        Self {
-            state: MessageState::Unspent,
-        }
-    }
-
-    /// Constructor for `MessageStatus` that fills with `Spent` state
-    pub fn spent() -> Self {
-        Self {
-            state: MessageState::Spent,
-        }
-    }
-
-    /// Constructor for `MessageStatus` that fills with `Unknown` state
-    pub fn not_found() -> Self {
-        Self {
-            state: MessageState::NotFound,
-        }
-    }
-}
-
-/// The possible states a Message can be in
-pub enum MessageState {
-    /// Message is still unspent
+pub enum MessageStatus {
```

Comment:
Ohh, I forgot that it affects the GraphQL API endpoint. Hmm, we need to find a way to still support this endpoint because the bridge relies on it.

Maybe we need to move this table to off-chain database instead.

Reply:
Hmmm in that case I'm probably just going to redo the whole PR.